static/js/common.js

```diff
@@ -62,6 +62,22 @@ exports.password_quality = function (password, bar, password_field) {
     return acceptable;
 };
 
+exports.password_warning = function (password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
+    if (password_field) {
+        min_password_length = password_field.data('minLength') || min_password_length;
```

Comment:
The change password form in `/#settings/your-account` don't have  data-min-length and data-min-quality attributes. So if we can't remove this now without adding them first. The account_settings.handlebar which has the password change form is rendered client side. So we  have to pass the value of min length and quality in page params to set the data-min-length and data-min-quality attributes. Should I do it that way?